dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -67,10 +67,23 @@ endpoint.httpOptions=HTTP Options
 endpoint.httpOptions$Ref=HTTP options reference
 endpoint.httpOptions.desc=HTTP protocol options for the endpoint.
 
+endpoint.remoteIp=Remote Ip Options
+endpoint.remoteIp$Ref=Remote ip options reference
+endpoint.remoteIp.desc=Remote ip options for the endpoint.
+
 endpoint.accessLog=HTTP Access Logging
 endpoint.accessLog$Ref=HTTP access logging reference
 endpoint.accessLog.desc=HTTP access logging configuration for the endpoint.
 
+http.remoteip=Remote Ip Options
+http.remoteip.desc=Remote Ip configuration
+
+http.remoteip.proxies=Proxies regex
+http.remoteip.proxies.desc=Regular expression that defines trusted proxies.
+
+http.remoteip.useRemoteIpInAccessLog=Use remote ip in access log
+http.remoteip.useRemoteIpInAccessLog.desc=If true, and the remote client information has been verified by the HTTP Channel, the NCSA Access Log will reflect the Forwarded or X-Forwarded-* headers when recording the remote client Ip, host, and/or the request protocol.
```

Comment:
Change to: If this property is set to true and the remote client information was verified by the HTTP channel, the NCSA access log reflects the Forwarded or X-Forwarded-* headers. These headers are reflected when one or more of the following items are recorded: the remote client, the IP, the host, or the request protocol.